README.md

```diff
@@ -30,31 +30,29 @@ An example `ICU4X` powered application in Rust may look like below...
 
```

Comment:
Reminder to update the new example in #3928 to fall back from `sr-ME` to `sr-Latn` which is a bit more interesting than the current example that only strips the region subtag.